server.go

```diff
@@ -883,21 +897,7 @@ func (s *Server) handleStream(t transport.ServerTransport, stream *transport.Str
 	method := sm[pos+1:]
 	srv, ok := s.m[service]
 	if !ok {
-		if trInfo != nil {
-			trInfo.tr.LazyLog(&fmtStringer{"Unknown service %v", []interface{}{service}}, true)
```

Comment:
Just out of curiosity, is there a gRPC-standard requirement for the description sent as part of these errors (i.e. is it mandate that you need to diff between method and service existence?)